testing/tests/DevExpress.ui.widgets/fileManagerParts/editing.tests.js

```diff
@@ -0,0 +1,293 @@
+
+/* global internals, createTestFileSystem */
+
+import $ from "jquery";
+import "ui/file_manager";
+import fx from "animation/fx";
+
+const { test } = QUnit;
+
+const moduleConfig = {
+    beforeEach: () => {
+        var fileSystem = createTestFileSystem();
+
+        this.clock = sinon.useFakeTimers();
+        fx.off = true;
+
+        $("#fileManager").dxFileManager({
+            fileSystemStore: fileSystem,
+            itemList: {
+                showFolders: false
+            }
+        });
+
+        this.clock.tick(400);
+    },
+
+    afterEach: () => {
+        this.clock.tick(5000);
+
+        this.clock.restore();
+        fx.off = false;
+    }
+};
+
+QUnit.module("Editing operations", moduleConfig, () => {
+
+    test("rename folder in folders area", (assert) => {
+        var $folderNode = this.$element.find(`.${internals.FOLDERS_TREE_VIEW_ITEM_CLASS}`).eq(0);
```

Comment:
Not use the `var`  in the file with ES6

Reply:
All the `var` have been removed from the code.